voice/methods/calls/about.md

```diff
@@ -9,7 +9,7 @@ Read More about Calls and Voice in the <a href="http://dev.bandwidth.com/faq/#vo
 
 ### Base URL
 
-`https://voice.bandwidth.com/api/v2/accounts/{accountId}`
+`https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls`
```

Comment:
I don't think anybody would reasonably interpret it this way, but I think it's technical valid to assume that endpoint = base + path, so you could interpret the docs as saying that `GET /calls` for example is at `https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/calls`.
Dunno how to best fix this; just a thought.
The same applies to the other resources' endpoints as well.

Reply:
yeah, I was thinking of this as the base url for the calls resource. 
The API base url is already in `voice/methods/about.md`
Who ever created the recordings part, thought the same since it has the `recordings` resource base url
What do others think? @Bandwidth/cash @Bandwidth/sinatra

Reply:
I would prefer to stay with whatever we're doing for this change